python/function.py

```diff
@@ -2603,6 +2684,13 @@ def set_user_var_value(self, var, def_addr, value):
 		core.BNSetUserVariableValue(self.handle, var_data, def_site, value._to_api_object())
 
 	def clear_user_var_value(self, var, def_addr):
+		"""
+		Clears a perviously informed user variable value.
```

Comment:
Typo. Also, for consistence maybe "defined user variable value" instead of "informed"?